pandas/tests/test_algos.py

```diff
@@ -246,9 +246,16 @@ def test_value_counts(self):
         factor = cut(arr, 4)
 
         tm.assertIsInstance(factor, Categorical)
-
         result = algos.value_counts(factor)
-        expected = algos.value_counts(np.asarray(factor))
+        # XXX: refactoring the test!
```

Comment:
So I had to change this test, but it was funky before. in the `assert_equal` below we call `.sort_index`. Before the index was just an Index, so the sorting was alphabetically. Now it's a `CategoricalIndex` so the sort order is the numerical.